### PR TITLE
Adsense support

### PR DESCRIPTION
**Pull request by javisabalete**
## Description
Adsense example code provided following analytics example code

## Motivation and Context

Many websites are using ads to fund themselves. It's interesting to have a space for them.

## How Has This Been Tested?

prod instance

## Screenshots (if appropriate):
screen shot 2018-01-27 at 00 37 16

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -102,6 +102,16 @@
 		</nav>
 
 		<div class="container">
+			<div>
+				<center>
+					<?php
+					// Google Adsense
+					if (is_file("adsense.php")) {
```
Comment (Obihoernchen) [inline index.php]: Please move the if up. We don't need the additional div if adsense is not used.

**Review by Obihoernchen — CHANGES_REQUESTED** [review]

**Comment by michikrug** [comment]
I would rather recommend to use the page_level_ads that are automatically inserted in all the pages by including the adsense script (like stated below) only in the head.
Same applies for analytics.

```
<script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<script>(adsbygoogle = window.adsbygoogle || []).push({ google_ad_client: "ca-pub-xxx", enable_page_level_ads: true });</script>
```

**Comment by javisabalete** [comment]
Page level ads is only for mobile browsers, however this PR show banners at desktop and mobile browsers

**Comment by michikrug** [comment]
This is not true at all. Page level ads perfectly serve banner ads also on desktop.

https://support.google.com/adsense/answer/6245304?hl=en

**Comment by javisabalete** [comment]
captura de pantalla 2018-01-28 a las 0 16 07

Chrome last version (MacOS)

**Comment by michikrug** [comment]
This is perfectly true, because the special type of anchor ads is only for mobile browsers.
But page level ads support all types of ads and choose those which fit the best.

bildschirmfoto 2018-01-28 um 00 23 36

I am using this now since a year. And it works great on both mobile and desktop.

**Comment by javisabalete** [comment]
Ooops, I can't see it

screenshot from 2018-01-30 03-40-39

:1st_place_medal:

**Comment by michikrug** [comment]
Oh. This is strange. I honestly have no idea why you do not have the switch.

Only on my map I use a fixed ad slot.

**Comment by Obihoernchen** [comment]
@javisabalete please check if we can use page level ads? Maybe there is an option you have to activate?

**Comment by michikrug** [comment]
The newest stuff is called "Auto ads" and will replace Page-level ads.
They are included/used exactly as page-level ads.

https://support.google.com/adsense/answer/7478040?hl=en#

**Pull request closed without merging**